### PR TITLE
Modify Labeler

### PR DESCRIPTION
**Pull request by louis1706**
## Description
**Describe the changes** 

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?** (if this is a feature change)

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Other information**:

<br />

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [ ] I have checked the project can be compiled
- [ ] I have tested my changes and it worked as expected

### Patches (if there are any changes related to Harmony patches)
- [ ] I have checked no IL patching errors in the console

### Other
- [ ] Still requires more testing

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,40 +1,36 @@
...
   - any-glob-to-any-file: EXILED/Exiled.Events/** # Any Modifications to events
+  - all-globs-to-all-files: '!EXILED/Exiled.Events/Patches/**/*' # Exept modifications to transpiler files
 
-regarding-api: # Add the 'API' label
+API: # Add the 'API' label
 - changed-files:
   - any-glob-to-any-file: EXILED/Exiled.API/** # Any modifications to the API
 
-regarding-transpiler: # Add the 'transpiler' label
+Transpiler: # Add the 'transpiler' label
```
Comment (ALEXWARELLC) [inline .github/labeler.yml:22]: Why is this being modified?

The names are the labels which were set for consistency with older labels.

File: .github/labeler.yml [hunk 2]
```diff
@@ -1,40 +1,36 @@
 Documentation:
 - changed-files:
-  - any-glob-to-any-file: docs/**
-
-NuGet:
-- changed-files:
-  - any-glob-to-any-file: '**/*.nuspec'
```
Comment (ALEXWARELLC) [inline .github/labeler.yml:7]: This is NuGet related so it should stay in NuGet, not be added to GitHub.

File: .github/labeler.yml [hunk 3]
```diff
@@ -1,40 +1,36 @@
...
-  - any-glob-to-any-file: EXILED/Exiled.CustomModules/** # Any modifications to CustomModules
+  - any-glob-to-any-file: EXILED/Exiled.Custom*/** # Any modifications to Exiled.Custom* file
 
 Installer: # Add the 'Installer' label
 - changed-files:
   - any-glob-to-any-file: EXILED/Exiled.Installer/** # Any modifications to the Installer
-
-Localization: # Add the 'Localization' label
-- changed-files:
-  - any-glob-to-any-file: EXILED/Localization/** # Any modifications to Localization
```
Comment (ALEXWARELLC) [inline .github/labeler.yml:36]: Does Exiled no longer do localised README?

File: .github/labeler.yml (hunk 1 above)
Comment (louis1706) [reply .github/labeler.yml:22]: We are going to modify label name after this merge

File: .github/labeler.yml (hunk 2 above)
Comment (louis1706) [reply .github/labeler.yml:7]: It's to reduce the number of label used 
There no need to differentiate Github and Script

File: .github/labeler.yml (hunk 3 above)
Comment (louis1706) [reply .github/labeler.yml:36]: It's now put under Documentation

**Review by ALEXWARELLC — APPROVED** [review]
All good, this is ok.

**Review by Vladislav-CS — APPROVED** [review]

File: .github/labeler.yml (hunk 1 above)
Comment (Misaka-ZeroTwo) [inline .github/labeler.yml:22]: Harmony-patch better? idk we still have prefix and postfix.

File: .github/labeler.yml [hunk 4]
```diff
@@ -1,40 +1,36 @@
...
+  - any-glob-to-any-file: '**/*.nuspec' #NuGet
   - any-glob-to-any-file: '**/*.ps1' # Windows
   - any-glob-to-any-file: '**/*.sh' # Linux
+  - any-glob-to-any-file: .github/** # Any modifications to github related files
 
-regarding-events: # Add the 'Events' label
+Events: # Add the 'Events' label
 - changed-files:
   - any-glob-to-any-file: EXILED/Exiled.Events/** # Any Modifications to events
+  - all-globs-to-all-files: '!EXILED/Exiled.Events/Patches/**/*' # Exept modifications to transpiler files
```
Comment (Misaka-ZeroTwo) [inline .github/labeler.yml:16]: Not sure why you felt patch changes were not part of the event but okay.

Comment (ALEXWARELLC) [reply .github/labeler.yml:16]: I didn't see this change, but while I think what Yamato is trying to is seperate transpires and events into isolated labels, the intent for the labeled was to help provide a quick way of understanding what parts of Exiled were actually being modified.

File: .github/labeler.yml (hunk 1 above)
Comment (louis1706) [reply .github/labeler.yml:22]: Idk how to make than only Transpiler patch are tagged

**Review by Jesus-QC — APPROVED** [review]

**Review by Jesus-QC — APPROVED** [review]

**Pull request merged**